scripts/add-preview-url-to-cloud-idp.ts

```diff
@@ -11,21 +11,26 @@ const fetch = createFetch()
 const vercelApiToken = process.env.VERCEL_API_TOKEN
 const vercelTeamId = process.env.VERCEL_TEAM_ID
 
+// For Vercel API docs
+// see https://vercel.com/docs/rest-api/endpoints#get-a-deployment-by-id-or-url
+
 /** A helper to fetch Vercel Deployment Aliases from Vercel Deploy Preview URLs */
 async function fetchDeploymentAlias(previewUrl: string): Promise<string[]> {
+	console.log('fetchDeploymentAlias', previewUrl)
+
 	const previewHostname = new URL(previewUrl).hostname
 	const url = new URL(
 		`v13/deployments/${previewHostname}`,
 		'https://api.vercel.com'
 	)
 
 	url.searchParams.set('teamId', vercelTeamId)
-	url.searchParams.set('withAutomaticAliases', '1')
 
 	const headers = { Authorization: `Bearer ${vercelApiToken}` }
 
 	const res = await fetch(url.toString(), { headers })
 	const data = await res.json()
+	console.log(data)
```

Comment:
🔪

Reply:
@dstaley I actually. need this log statement :P